Define initial traffic-generator.yang model

purplezimmermann opened the pull request:
And add dependency `ietf-yang-types@2013-07-15.yang`

purplezimmermann committed:
Define initial traffic-generator.yang model

vlvassilev reviewed (dismissed):
Maybe skip standard models.

vlvassilev commented on yang/traffic-generator.yang:
```diff
@@ -0,0 +1,49 @@
+        "Source MAC address in ethernet frame headers.";
+    }
+
+    leaf destination-address {
```
MAybe shorten destination to dst.

purplezimmermann replied:
@vlvassilev: Addressed! And also shortened `source-address` to `src-address`

purplezimmermann commented:
@vlvassilev: Deleted `ietf-yang-types@2013-07-15.yang` from the commit

purplezimmermann commented:
@vlvassilev 

* Added `description` and first `revision`
* Removed info about `ietf-yang-types@2013-07-15.yang` from commit message
* Validated module with `pyang --lint`
